builtin/providers/dme/resource_dme_record_test.go

```diff
@@ -36,6 +36,9 @@ func TestAccDMERecord_basic(t *testing.T) {
 						"dme_record.test", "value", "1.1.1.1"),
 					resource.TestCheckResourceAttr(
 						"dme_record.test", "ttl", "2000"),
+                                        resource.TestCheckResourceAttr(
+                                                "dme_record.test", "gtdLocation", "DEFAULT"),
```

Comment:
you only test it with 'DEFAULT'? Is there a testcase that you can make where you actually to something more useful?